test/api/qcFlagTypes.test.js

```diff
@@ -415,4 +416,105 @@ module.exports = () => {
                 });
         });
     });
+
+    describe('POST /api/qcFlagTypes', () => {
+        it('should successfuly create QC Flag Type', (done) => {
+            const parameters = {
+                name: 'A',
+                method: 'AA+',
+                bad: false,
+                color: '#FFAA00',
+            };
+
+            request(server)
+                .post('/api/qcFlagTypes?token=admin')
+                .send(parameters)
+                .expect(201)
+                .end((err, res) => {
+                    if (err) {
+                        done(err);
+                        return;
+                    }
+
+                    const { data: newQCFlag } = res.body;
+                    expectObjectToBeSuperset(newQCFlag, parameters);
+                    done();
+                });
```

Comment:
Maybe check that the flag has actually been created?

Reply:
added